### PR TITLE
Add browserconfig.xml

### PR DESCRIPTION
**Pull request by myxrome**

### CODE REVIEW

**Comment by myxrome** [comment]
[[TARIFF17] Add a browserconfig.xm](https://trello.com/c/DEU6tecj/538-tariff17-add-a-browserconfigxm)

**Comment by alexey-klimuk** [comment]
@matthewford should we add here something?

File: public/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+  <msapplication>
+    <tile>
+      <TileColor>#005ea5</TileColor>
+    </tile>
+  </msapplication>
+</browserconfig>
```
Comment (alexey-klimuk) [inline public/browserconfig.xml]: @myxrome I think we need empty lines at the end of each file

**Comment by myxrome** [comment]
@alexey-klimuk @matthewford I checked the [browserconfig.xml specification](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/dn320426(v=vs.85)). The aim of this file:

> define pinned site customizations, such as tile backgrounds, badge updates, and tile notifications

It's mostly related to frontend app, please check the related PRs: [trade-tariff-frontend](https://github.com/bitzesty/trade-tariff-frontend/pull/208) and [trade-tariff-admin](https://github.com/bitzesty/trade-tariff-admin/pull/89).

We can define the site icon with different sizes for tiles **used in Windows 10 only**, but we have to add those icons to public/. Therefore we haven't icons, I think that it's ok to define tile background color only.

**Comment by matthewford** [comment]
oh wait @myxrome this needs to be on the frontend

**Comment by myxrome** [comment]
@matthewford see [trade-tariff-frontend](https://github.com/bitzesty/trade-tariff-frontend/pull/208). I added browserconfig.xml to all projects to exclude any chance of 404.

**Pull request merged**